src/plugins/intel_gpu/tests/unit/test_cases/pooling_gpu_test.cpp

```diff
@@ -2197,7 +2197,8 @@ INSTANTIATE_TEST_SUITE_P(
                                      format::bfyx,
                                      format::b_fs_yx_fsv4,
                                      format::b_fs_yx_fsv16,
-                                     format::b_fs_yx_fsv32)),
+                                     format::b_fs_yx_fsv32,
+                                     format::fs_b_yx_fsv32)),
```

Comment:
how many test case does this create? I'm afraid this will generate too much case for rare usage. I'd suggest to introduce separate test instantiation.

Reply:
Updated tests with separate test instantiation. 16 tests and 0.2 sec to run this.